src/transaction/transaction.spec.ts

```diff
@@ -441,9 +441,13 @@ describe('Resource Packer: meta', () => {
   })
 
   test('error during simulate', async () => {
-    await expect(externalClient.send.call({ method: 'error' })).rejects.toThrow(
-      'Error during resource population simulation in transaction 0',
-    )
+    try {
+      await externalClient.send.call({ method: 'error' })
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } catch (e: any) {
+      expect(e.stack).toMatch(`err <--- Error`)
```

Comment:
Just noting that this was working before this change. Just wanted to make the test more explicit to catch any future regressions